### ISSUE TITLE
[Data] Null Values In Company API

### ISSUE THREAD

**Issue by rmantovani** [iexg/IEX-API#1660]
**Clearly describe the data problem**
many fields are coming with null values (industry, website, description, CEO, employees)

**Data URL (The URL you used to access the data without your token):**
https://cloud.iexapis.com/stable/stock/dis/company?token=XYZ

**What time did the problem occur (including timezone)?**
Any time

**What is the expected behavior?**
Correct field values as usual

**What are the steps to reproduce**
call company api, symbol DIS, XP and many others

**Your environment (e.g. browser, OS, API version, language):**
Linux Java

**Additional context**

{"symbol":"DIS","companyName":"Walt Disney Co (The)","exchange":"NEW YORK STOCK EXCHANGE, INC.","industry":"","website":"","description":"","CEO":"","securityName":"Walt Disney Co (The)","issueType":"cs","sector":"","primarySicCode":7990,"employees":null,"tags":["Consumer Services","Media Conglomerates"],"address":null,"address2":null,"state":null,"city":null,"zip":null,"country":null,"phone":null}

**Comment by adamwozney-iex** [comment]
Hey @rmantovani - We recently updated our READme file and moved all support related questions to another channel. Please reach out to support@iexcloud.io and our support team will follow-up with you there. Sorry for any confusion.

Thanks.

**Comment by rmantovani** [comment]
@adamwozney-iex  this is a bug, the values are missing.

**Comment by StockConsultant** [comment]
Yes I have also noticed they have been missing for a lot stocks especially ETFs since 12/1 update

**Comment by jesseorosz** [comment]
@rmantovani 

This is an upgrade!!!!!!

**Comment by whoisal** [comment]
The issue is still there with most (if not all) companies returning an empty string in description.

**Comment by StockConsultant** [comment]
It really shows up on all the etfs

________________________________________

**Comment by adamwozney-iex** [comment]
Hey @rmantovani, @StockConsultant, and @whoisal - Thanks for flagging. 

If you could email our support team at support@iexcloud.io with your email address that's associated with your IEX Cloud account and a link to this post, our support team can review and follow-up with you. We're able  
 to better serve you by tracking support requests and issues that come up this way. Thanks!

**Comment by timpaine-iex** [comment]
closing into https://github.com/iexg/IEX-API/issues/1616
